src/lib/Microsoft.Health.Fhir.Ingest.Template/InvalidTemplateException.cs

```diff
@@ -39,5 +41,9 @@ public override string Message
         public ILineInfo GetLineInfo => _lineInfo;
 
         public bool HasLineInfo => _lineInfo.HasLineInfo();
+
+        public override string ErrSource => nameof(ErrorSource.User);

```

Comment:
How is this being used?  It seems like a one off.  I would think if it is used and important I should see this being set in more exceptions beyond this one.